### BIG differences, this is diff of wrong files.
### CLUS "Variable /, //, ///"
### and CLHS topic selection, I don't find it
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:variables:repl-slash.txt	2017-04-27 21:58:35.000000000 +0100
+++ saved-pages-clus/cl:variables:repl-slash.txt	2017-04-27 21:58:35.000000000 +0100
@@ -1,4 +1,64 @@
-/
-Please select which reference to / you intended
+Variable /, //, ///

-FunctionVariable
\ No newline at end of file
+Value Type
+a proper list.
+Initial Value
+implementation-dependent.
+Description
+variables /, //, and /// are maintained by the Lisp read-eval-print loop to save the values of results that were printed at the end of the loop.
+The value of / is a list of the most recent values that were printed, the value of // is the previous value of /, and the value of /// is the previous value of //.
+The values of /, //, and /// are updated immediately prior to printing the return value of a top-level form by the Lisp read-eval-print loop. If the evaluation of such a form is aborted prior to its normal return, the values of /, //, and /// are not updated.
+Examples
+(floor 22 7)
+→
+3
+1
+(+ (* (car /) 7) (cadr /))
+→
+22
+Affected By
+Lisp read-eval-print loop.
+See Also
+-, +, *, {\secref\TopLevelLoop}
+Notes
+None.
\ No newline at end of file
